src/screens/GalleryScreen/GalleryScreen.tsx

```diff
@@ -41,16 +50,21 @@ export const GalleryScreen: React.FC = () => {
     try {
       const videoArrayFromLocalStorage = await AsyncStorage.getItem(videoKey);
       if (videoArrayFromLocalStorage === null) {
-        Alert.alert('No videos from local storage');
+        Toast.show({
+          type: 'error',
+          text1: "There aren't any videos to show. Go take one! ",
+        });
         return;
       }
       const videoArrayJSON: VideoFile[] = JSON.parse(
         videoArrayFromLocalStorage,
       );
       setVideoArrayState(videoArrayJSON);
     } catch (error) {
-      Alert.alert('Something went wrong at VideoScreen VideoJSON');
-      console.warn(error);
+      Toast.show({
+        type: 'error',
+        text1: GENERIC_ERROR_MESSAGE,
+      });
```

Comment:
Again with the generic error messages, probably not a good idea in the long run.